sources/packages/backend/libs/sims-db/src/entities/education-program-offering.model.ts

```diff
@@ -18,7 +18,7 @@ import { numericTransformer } from "@sims/sims-db/transformers/numeric.transform
  * Max value the offering name can have. By DB definition it is defined as
  * 300 but from business perspective it should not be longer than 50.
  */
-export const OFFERING_NAME_MAX_LENGTH = 50;
+export const OFFERING_NAME_MAX_LENGTH = 100;
```

Comment:
Please adjust the comment. `from business perspective it should not be longer than 100`